### ISSUE TITLE
Accuracy only ~70% on ucf101.

### ISSUE THREAD

**Issue by bupt-wcm** [KaiyuYue/cgnl-network.pytorch#6]
Thanks for the great job. I follow the training strategy in the paper to train a I3DResNet50 on ucf101, and the ImageNet pretrained model is used.  I sample 64 consecutive frames and drop evenly as the training input and sample 30x32 frames as the testing input. I3DResNet is converted from C2D mentioned in Non-local network.
However, I can only get about 70% accuracy.  So, can you provide the script about the task of video classification or give some suggestions? Thank you.

**Comment by KaiyuYue** [comment]
Hi @WangCMing , please check out the tips in the previous [issue](https://github.com/KaiyuYue/cgnl-network.pytorch/issues/4#issuecomment-481606712). 

Assume your training details has been kept same as those in [Non-Local Network](https://arxiv.org/abs/1711.07971), please double check your inference method. Fully convolutional inference is very important to benchmark the video recognition models. 

Recently I notice a related [work](https://github.com/HolmesShuan/Compact-Global-Descriptor) (Compact-Global-Descriptor) which report the results on UCF101 using ResNet50 + CGNL Module, hope it can help you.

**Comment by kaiyuyue** [comment]
Hi @WangCMing , please check out the tips in the previous [issue](https://github.com/KaiyuYue/cgnl-network.pytorch/issues/4#issuecomment-481606712). 

Assume your training details has been kept same as those in [Non-Local Network](https://arxiv.org/abs/1711.07971), please double check your inference method. Fully convolutional inference is very important to benchmark the video recognition models. 

Recently I notice a related [work](https://github.com/HolmesShuan/Compact-Global-Descriptor) (Compact-Global-Descriptor) which report the results on UCF101 using ResNet50 + CGNL Module, hope it could help.
